javalib/src/main/scala/java/util/AbstractList.scala

```diff
@@ -11,6 +11,10 @@ abstract class AbstractList[E] protected ()
     true
   }
 
+  // tests/compile:nativeLinkNIR fails without this re-declaration for some reason
+  // cannot link: @java.util.AbstractList::get_i32_java.lang.Object
```

Comment:
This probably another instance of https://github.com/scala-native/scala-native/issues/375

Please cite this issue in the comment.